appshell/cef_dark_aero_window.cpp

```diff
@@ -816,7 +816,7 @@ LRESULT cef_dark_aero_window::WindowProc(UINT message, WPARAM wParam, LPARAM lPa
         if (mReady) {
             UpdateNonClientArea();
         }
-        break;
+        return 0L;
```

Comment:
Seems to work ok but the app frame is no longer drawing in the active color when activated. 
My guess is that Windows isn't sending `WM_ACTIVATEAPP` because you're preventing the `WM_ACTIVATE` message from reaching the default Window Proc but I haven't debugged this to determine if that is indeed the case.

```C
    case WM_ACTIVATEAPP:
        mIsActive = (BOOL)wParam;
        UpdateNonClientArea();
        break;
    }
    return lr;
```

I guess technically you could just keep track of the active state from here but I wonder if there are other things that are broken.